bot/processors/pennychat.py

```diff
@@ -0,0 +1,289 @@
+                    "text": "Description"
+                },
+                "hint": {
+                    "type": "plain_text",
+                    "text": 'Give people an idea of what this chat will be about.'
+                }
+            },
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "*Pick a date for the Penny Chat*"
+                }
+            },
+            {
+                "block_id": "penny_chat_date_time",
+                "type": "actions",
+                "elements": [
+                    {
+                        "type": "datepicker",
+                        "action_id": "penny_chat_date",
+                        "initial_date": date,
+                    },
+                    {
+                        "type": "static_select",
+                        "action_id": "penny_chat_time",
+                        "initial_option": time,
+                        "options": time_options
+                    }
+                ]
+            },
+            {
+                "block_id": "penny_chat_user_select",
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "*Who do you want to invite?*"
+                },
+                "accessory": {
+                    "type": "multi_users_select",
+                    "placeholder": {
+                        "type": "plain_text",
+                        "text": "Select Users"
+                    },
+                    "initial_users": users,
+                    "action_id": "penny_chat_user_select",
+                }
+            },
+            {
+                "block_id": "penny_chat_channel_select",
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "*Which channels should I share this with?*"
+                },
+                "accessory": {
+                    "type": "multi_channels_select",
+                    "placeholder": {
+                        "type": "plain_text",
+                        "text": "Select Channels"
+                    },
+                    "initial_channels": channels,
+                    "action_id": "penny_chat_channel_select",
+                }
+            },
+        ]
+    }
+
+    return template
+
+
+class PennyChatBotModule(BotModule):
+    def __init__(self, slack):
+        self.slack = slack
+
+    @classmethod
+    def create_penny_chat(cls, slack, event):
+        response = slack.chat_postMessage(channel=event['channel_id'],
+                                          user=event['user_id'],
+                                          blocks=create_penny_chat_blocks()
+                                          )
+
+        PennyChat.objects.create(template_channel=response.data['channel'], template_timestamp=response.data['ts'])
```

Comment:
how do we get this object once it's created? I see a call below to `PennyChat.objects.get(template_timestamp=event['container']['message_ts'])` but will that be a dependable way to retrieve the correct object? What about when we're popular and multiple penny chats are scheduled every second? :D